spec/services/calculators/dependant_allowance_calculator_spec.rb

```diff
@@ -221,25 +230,25 @@ module Calculators
 
         describe "child_under_15_allowance" do
           it "returns the threshold value" do
-            expect(calculator.child_under_15_allowance).to eq 298.08
+            expect(calculator.send(:child_under_15_allowance)).to eq 298.08
```

Comment:
these changes look unnecessary

Reply:
Ay, yes, so: When you recommended I make `attr_reader :submission_date` in DependantAllowanceCalculator private I had a look, and saw that there were a bunch of other internal methods that should be private in that calculator. So I made them private. Then I saw that this spec was calling a bunch of those methods, as a way of testing the values in the Threshold yml files. So with those methods now marked private, these specs were failing.

TBH, these specs feel pretty dodgy to me. So rather than pretend that those private methods were actually supposed to be part of the public interface of the class, I decided to update the tests to make it more visible that what they're doing is a bit suspect.